vendor/ansible_collections/nvidia/bare_metal/plugins/modules/instance.py

```
@@ -321,6 +328,7 @@
 operating_system_id=dict(type='str'),
 phone_home_enabled=dict(type='bool'),
 reboot_with_custom_ipxe=dict(type='bool'),
+secondary_vpc_ids=dict(type='list', elements='str'),
```

Flake8 (7.3.0)

[error] continuation line missing indentation or outdented

(E122)